source/includes/_using.md

```diff
@@ -64,27 +74,23 @@ Inspetor provides additional methods to allow you to inform us about meaningful
-
-In the future however, our suggested integration model will be:
+The answer is in as many places as possible. From your frontend interfaces (iOS/Android apps, JS websites), we're able to derive valuable information such as device characteristics, user behavior, and IP information, while from the backend you can provide us with a more holistic representation of your data in terms of business logic. Both of these types of data are extremely valuable for detecting fraud. As such, we suggest the following:
 
 - Full event coverage within the server/API level of your application (backend): This allows us to know everything about what is happening within your product.
 - 1:1 corresponding coverage within your frontend interfaces (Web, iOS, and Android): This allows us to enrich the knowledge we're gathering from your server-level transactions with additional information that improves our protection capabilities.
 
-You can find additional language-specific implementation details, including suggested architecture and best practices, via the following links:
+Note that the information that you provide in either environment will vary. In the frontend integrations, you can rely on our SDKs to automatically relay the majority of the information we use to make a decision. You can read more about integrating with our frontend SDKs [here](https://inspetor.github.io/docs-frontend).
 
-- [PHP](https://github.com/inspetor/libraries-docs/blob/master/backend/php/README.md)
-- [Python](https://github.com/inspetor/libraries-docs/blob/master/backend/python/README.md)
 
 ## Testing your integration with Inspetor
 
-Integrating with Inspetor is meant to be easy--simply instantiate our library, call our tracking methods, and if there are no errors thrown within your code execution, data will appear within our database. However, for further validation, we also provide our customers with limited-access database credentials that allow the developer in charge of integrating Inspetor to inspect that data appears in our database. (Note that access is restricted such that your customer account will only be able to view data originating from your company's integration.) The Inspetor team will provide you with access credentials for this phase of validation.
+Integrating with Inspetor is meant to be easy--simply instantiate our library or hit our REST API with valid authentication, and if there are no errors thrown within your code execution, data will appear within our database. For further validation, we also provide our customers with limited-access database credentials that allow the developer in charge of integrating Inspetor to inspect that data appears in our database. (Note that access is restricted such that your customer account will only be able to view data originating from your company's integration.) The Inspetor team will provide you with access credentials for this phase of validation.
```

Comment:
So we expect our clients (Ingresse, Singu, etc) to have to check in our DB for the data? I think that's not the most user-friendly approach to integration. This would mean that they have to know which tables to check, and this is more related to our "business logic" side than theirs. I believe, in terms of sandbox integration, that we should be able to have a way for them to integrate, check that they are getting expected responses on the API calls, and then perhaps we should also provide some `get` endpoints for them to be able to run some checks, but being very clear about how they would use that (ie "first, you send us a `post` request in our `xyz` endpoint. Then, you can send a `get` at endpoint `abc`, expecting to find infos `foo`, `bar` and `baz` in the response for this request")

Reply:
This is a great point. The way that a user should be able to validate that they're interacting with our API correctly is by HTTP response statuses. 

This documentation was written for the SDK world (hence the somewhat convoluted validation procedure). I'll update :)